docs/content/docs/connectors/table/jdbc.md

```diff
@@ -58,7 +73,8 @@ A driver dependency is also required to connect to a specified database. Here ar
 | OceanBase  | `com.oceanbase`            | `oceanbase-client`     | [Download](https://repo1.maven.org/maven2/com/oceanbase/oceanbase-client/)                                                        |
 
 
-JDBC connector and drivers are not part of Flink's binary distribution. See how to link with them for cluster execution [here]({{< ref "docs/dev/configuration/overview" >}}).
+JDBC connector and drivers are not part of Flink's binary distribution. 
+See how to link with them for cluster execution [here]({{< ref "docs/dev/configuration/overview" >}}).
```

Comment:
nit: what do we mean by `link with`. It sounds like linking after compiling. I suggest, `See how to use them`